bravado/client.py

```diff
@@ -148,10 +148,7 @@ def from_spec(cls, spec_dict, origin_url=None, http_client=None,
     def get_model(self, model_name):
         return self.swagger_spec.definitions[model_name]
 
-    def __repr__(self):
-        return u"%s(%s)" % (self.__class__.__name__, self.swagger_spec.api_url)
-
-    def __getattr__(self, item):
+    def get_resource(self, item):
```

Comment:
This could in theory break users that have a tag of "get_resource". Obviously we didn't worry about this back when this class was written, as we also have other methods that don't start with an underscore. That said, I wonder if we should start following that practice now. It's not consistent (which is not great), but still seems like the right thing to do.

Could you please rename the method to "_get_resource"? We do this for the Model class as well.

Reply:
Done.